arbos/l1pricing/l1pricing.go

```diff
@@ -37,10 +37,11 @@ type L1PricingState struct {
 	lastUpdateTime     storage.StorageBackedUint64 // timestamp of the last update from L1 that we processed
 	fundsDueForRewards storage.StorageBackedBigInt
 	// funds collected since update are recorded as the balance in account L1PricerFundsPoolAddress
-	unitsSinceUpdate storage.StorageBackedUint64 // calldata units collected for since last update
-	pricePerUnit     storage.StorageBackedBigInt // current price per calldata unit
-	lastSurplus      storage.StorageBackedBigInt // introduced in ArbOS version 2
-	perBatchGasCost  storage.StorageBackedBigInt // introduced in ArbOS version 2
+	unitsSinceUpdate   storage.StorageBackedUint64 // calldata units collected for since last update
+	pricePerUnit       storage.StorageBackedBigInt // current price per calldata unit
+	lastSurplus        storage.StorageBackedBigInt // introduced in ArbOS version 2
+	perBatchGasCost    storage.StorageBackedBigInt // introduced in ArbOS version 2
+	amortizedCostCapBP storage.StorageBackedUint64 // in basis points; introduced in ArbOS version 3
```

Comment:
This could be a `StorageBackedBips`, but the setter would have to error on negative values. Either sounds reasonable because of that